templates/nexus.properties.erb

```diff
@@ -13,3 +13,6 @@ nexus.clustered=<%= node['nexus_repository_manager']['properties']['clustered']
 <% if node['nexus_repository_manager']['properties']['license_file'] %>
 nexus.licenseFile=<%= node['nexus_repository_manager']['properties']['license_file'] %>
 <% end %>
+<% if node['nexus_repository_manager']['license_s3_bucket'] != '' %>
```

Comment:
I think this should be an else if condition to the license_file condition above.

Reply:
+1

Reply:
created 'elsif' (not a typo) in 6870a11